app/controllers/spina/journal/primer_theme/issues_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Spina
+  module Journal
+    module PrimerTheme
+      # User-facing controller for journal issues
+      class IssuesController < ApplicationController
+        before_action :set_journal
+        before_action :set_issue, :set_breadcrumb, only: :show
+        before_action :set_metadata
+
+        def index
+          # having multiple journals is not currently allowed anyway
+          @issues = Admin::Journal::Issue.sorted_desc
+        end
+
+        def show
+          add_breadcrumb t('spina.journal.primer_theme.volume_issue', volume_number: @issue.volume.number, issue_number: @issue.number)
+        end
+
+        private
+
+        def set_journal
+          @journal = Admin::Journal::Journal.instance
```

Comment:
I realise this is an awkward comment to make at this stage 😜 but I was wondering about the relative merits of using a slightly different schema if the journal has to be a singleton—in particular, I was wondering about getting the journal name from a preference stored with `Spina::Account` (there's existing support for this) and then just getting at the volumes directly from `Spina::Admin::Journal::Volume`. Just thinking that there's a lot of awkwardness with having singleton models (as I think you've experienced yourself) which would be nice to be able to avoid. But I'm missing a lot of context so this may not be possible—I guess it comes down to whether the journal has to have record-like functionality

Reply:
This is one of the earliest decisions I made and in hindsight probably not ideal - the original reason was that I was originally allowing for support for multiple journals. I later realised this was at minimum overcomplicating things for now, perhaps also overall unnecessary. But I couldn't bring myself to remove the record entirely, and worked out a singleton solution which I knew was bad but which allowed both possibilities to be a thing in future. It should be robust though, [here's the definition](https://github.com/louis-vs/spina-admin-journal/blob/4ba8e359b599305bed3b7a45386a82d086e92fa9/app/models/spina/admin/journal/journal.rb#L37).

Reply:
for instance, by the way - and I just thought of this - but the Conference Proceedings for ULAB could be stored as a journal, instead of duplicating everything in the conferences plugin (since not every conference paper goes into the proceedings). So there's definitely use cases for journal-as-record. It would require some refactoring, because I've lazily made the assumption in places that there is only one journal. But it shouldn't be too hard to release the flood gates on this in future.